packages/@aws-cdk/aws-codeartifact/test/domain.test.ts

```diff
@@ -0,0 +1,32 @@
+import '@aws-cdk/assert/jest';
+import { Stack } from '@aws-cdk/core';
+import { Domain } from '../lib';
+
+test('Create Domain', () => {
```

Comment:
Missing tests:
1. Any validation
2. Checking the generated CloudFormation for expected results.
3. Any tests related to the key and policy
4. Testing the grant methods.